### DISCUSSION TITLE
{% Disabling statements? %}

### DISCUSSION THREAD

**Discussion by rrauenza** [pallets/jinja#1912] [Q&A]
I'm thinking of limiting what my users can do in the initial release of my tool that uses jinja -- is there a way to disable (i.e. throw an error) in the parser when they try to use {% %} ?   

My tool's use case is more like how ansible uses "{{ foo }}" for variable expansion in .yml  rather than how ansible uses it to make templated configuration files.

Looking through the code, it doesn't look like I can't just set `block_start_string ` to "" or None (although setting it to uuid.uuid4() each run might work!)

**Comment by davidism** [comment]
Yes, you should be able to do this with an extension or an AST visitor. The docs have a tiny bit about using these, but you'll mostly have to explore and figure out what works for your needs.
